### DISCUSSION TITLE
Config Issues - Moving from WG to WGDashboard

### DISCUSSION THREAD

**Discussion by digital-knave** [WGDashboard/WGDashboard#503] [Q&A]
I've previously installed Wireguard on Proxmox using the script here which works like a charm:

https://github.com/Nyr/wireguard-install

Although I do have to make the /dev/net/tun config changes as outlined in this [proxmox wiki post here](https://pve.proxmox.com/wiki/OpenVPN_in_LXC).

Of course this doesn't have the nice WG-Dashboard front end. However despite the [Wireguard Helper Script from here](https://community-scripts.github.io/ProxmoxVE/scripts?id=wireguard) installing just fine and WG-Dashboard working too, I just cannot get it working when I try to connect to it via my phone.

I'm guessing it must be a config issue somewhere but I've even checked the .conf files from the working version against the WG-Dashboard version and things seem set up the same so I'm struggling to see what the issue is. 

Has anyone else struggled with this or managed to get it working?

Config created from WG-Dashboard looks like this:

```
[Interface]
Address = 10.7.0.1/24
PrivateKey = [redacted]
ListenPort = 51820

[Peer]
PublicKey = [redacted]
AllowedIPs - 10.7.0.2/32
```

Peer config (which I can only really find if I select option in the peer sharing methods) looks like this:

```
[Interface]
Address = 10.7.0.2/24
DNS = 8.8.8.8
PrivateKey = [redacted]

[Peer]
PublicKey = [redacted]
AllowedIPs = 0.0.0.0/0
Endpoint = my.duckdns.domain:51820
PersistentKeepalive = 25
```
Anyone have any thoughts? For now I've just gone back to the non-WGDashboard version which is working but doesn't give me the visibility I'd like.

**Comment by DaanSelen** [comment]
Carrying over information from Proxmox Helper Scripts, I am unsure to how the routing is done. In the new PR in that repository they use the standard way: https://github.com/community-scripts/ProxmoxVE/blob/fb0706469b27dc72661aa1fd77a9bbb5eb6f00dd/install/wireguard-install.sh#L46

So if you add the template, does anything change?
